pkg/exporter/grades.go

```diff
@@ -0,0 +1,47 @@
+package exporter
+
+import (
+	"github.com/anas-aso/ssllabs_exporter/pkg/ssllabs"
+)
+
+// Map grades to numerical values as defined in https://github.com/ssllabs/research/wiki/SSL-Server-Rating-Guide#methodology-overview
+// Since the documented mapping provides a range of values for each grade instead of fixed ones, we take half the documented interval
+// to allow mapping case like A+ and A-.
+// A special undocumented cases are T and M for which we assign 0.
+var gradesMapping = map[string]float64{
+	"A":     (80 + 100) / 2,
+	"A+":    ((80 + 100) / 2) + 1,
```

Comment:
nitpick: i'm not really sure about how the grades are mapped since it's not very intuitive E.g. `A+` would become a value of `91`  but a grade `A+` implies a value of `100` with exceptional secure settings.

Reply:
@svenmueller those values are internal to the exporter and not exposed. They help are being used currently to compare the grades and pick the "lowest" one.
I have updated the readme to explain what the exposed metrics mean and the expected values : https://github.com/anas-aso/ssllabs_exporter/pull/4/commits/486f7b54cbe60b1bbe1d1958a2d6937a23054f9c